src/tilesource.js

```diff
@@ -275,28 +276,39 @@ $.TileSource.prototype = {
         return this._tileHeight;
     },
 
+    /**
+     * @function
```

Comment:
I know the functions in this file aren't well-documented, but as long as you're in here, it might be nice to add just a little bit here in the doc comment about what this function does and why it exists.